skbio/stats/ordination/__init__.py

```diff
@@ -113,12 +113,13 @@
 from ._correspondence_analysis import CA
 from ._redundancy_analysis import RDA
 from ._canonical_correspondence_analysis import CCA
-from ._principal_coordinate_analysis import PCoA
+from ._principal_coordinate_analysis import PCoA, PCoABase
```

Comment:
I'm not sure we should import the base class here (nor export it in `*` imports).